File: PHP/time.php
<?php

function input(){
    return trim(fgets(STDIN));
}

$n = (int)input();
for ($i = 0; $i < $n; $i++){
    $num = (int)input();
    echo ($num % 100 < 60 && intdiv($num, 100) < 24) ? "YES" : "NO";
    echo PHP_EOL;
}

?>

File: Rust/time.rs
use std::io;

fn main() {
    let mut n = String::new();
    io::stdin().read_line(&mut n).unwrap();
    let n: i32 = n.trim().parse().unwrap();
    
    for _ in 0..n {
        let mut num = String::new();
        io::stdin().read_line(&mut num).unwrap();
        let num: i32 = num.trim().parse().unwrap();
        print!("{} ", if num % 100 < 60 && num / 100 < 24 { "YES" } else { "NO" });
    }
}